src/ansys/fluent/core/utils/search.py

```diff
@@ -138,18 +159,18 @@ def search(
     Parameters
     ----------
     word : str
-        The word to search for.
+        Word to search for.
     match_whole_word : bool, optional
         Whether to match whole word, by default False
     match_case : bool, optional
         Whether to match case, by default False
     version : str, optional
-        Fluent version to search in, by default None in which case
-        it will search in the latest version for which codegen was run.
+        Fluent version to search in. The default is ``None``. If ``None``,
+        it searches in the latest version for which codegen was run.
     search_root : Any, optional
-        The root object within which the search will be performed,
-        can be a session object or any API object within a session,
-        by default None in which case it will search everything.
+        The root object within which the search is performed,
+        can be a session object or any API object within a session.
```

Comment:
```suggestion
        The root object within which the search is performed.
        It can be a session object or any API object within a session.
```
This comma should be a full stop instead because what follows is a new sentence.

Reply:
Done. Thank you.